Language reference "compile variables" example doesn't work

### Zig Version

0.10.0-dev.4157+4961044ce

### Steps to Reproduce

Run the example from the documentation at https://ziglang.org/documentation/master/#Compile-Variables
```
const builtin = @import("builtin");

pub fn main() void {
    const separator = if (builtin.os.tag == builtin.Os.windows) '\\' else '/';
    _ = separator;
}
```
(I wrapped it in a main function and discarded the variable to make it a valid program)

### Expected Behavior

It compiles.

### Actual Behavior

```
r.zig:4:52: error: root struct of file 'builtin' has no member named 'Os'
    const separator = if (builtin.os.tag == builtin.Os.windows) '\\' else '/';
                                            ~~~~~~~^~~
referenced by:
    callMain: /home/yujiri/code/zig/build/stage3/lib/zig/std/start.zig:568:17
    initEventLoopAndCallMain: /home/yujiri/code/zig/build/stage3/lib/zig/std/start.zig:512:51
    remaining reference traces hidden; use '-freference-trace' to see all reference traces
```

I thought this might just be outdated due to a recent change, but even with zig 0.9.1, I get this error :think

--- a/doc/langref.html.in
+++ b/doc/langref.html.in
@@ -10768,7 +10768,7 @@ test "string literal to constant slice" {
       </p>
       {#code_begin|syntax#}
 const builtin = @import("builtin");
-const separator = if (builtin.os.tag == builtin.Os.windows) '\\' else '/';
+const separator = if (builtin.os.tag == .windows) '\\' else '/';
       {#code_end#}
       <p>
       Example of what is imported with {#syntax#}@import("builtin"){#endsyntax#}:

